scripts/cli.py

```diff
@@ -491,6 +491,13 @@ def set():
 def board(board):
     Config().set(Config().KEY_BOARD, board)
 
+
```

Comment:
On commit message: 
`Add 'set ot-path <OT-PATH>' command option that permits the user to` -> changes `permits` to `allows`
`The defined value will be saved at the config file alongside with with` -> Use just one `with`

Reply:
Fixed